racket/collects/racket/private/case.rkt

```diff
@@ -8,18 +8,27 @@
                          "stxcase-scheme.rkt"
                          "qqstx.rkt" "define.rkt" "sort.rkt" "fixnum.rkt"))
   (#%provide case)
+
+  ;; identifier? :: any/c -> boolean?
+  (define-for-syntax (identifier? stx)
```

Comment:
`identifier?` is available from `"stx.rkt"`, so it would be better to import that.

Reply:
Thanks, fixed.